py/desispec/io/frame.py

```diff
@@ -91,6 +91,19 @@ def write_frame(outfile, frame, header=None, fibermap=None, units=None):
 
     return outfile
 
+def read_meta_frame(filename):
+    """ 
+    Args:
+        filename: path to a file, or (night, expid, camera) tuple where
```

Comment:
I prefer how we have the I/O module handle most/all
of the I/O.  Could be we'll change the format of saving
Frame files (e.g hdf5), although I suppose that isn't too likely.

Anyhow, I've kept this and made the requested changes.